.github/workflows/daily.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Reclaim Disk Space
         run: .github/ci-prerequisites.sh
       - name: Install JDK {{ matrix.java }}
-        uses: joschi/setup-jdk@e87a7cec853d2dd7066adf837fe12bf0f3d45e52
+        uses: joschi/setup-jdk@v2
```

Comment:
I think we should switch to use [`actions/setup-java`](https://github.com/actions/setup-java) GHA instead, since [`joschi/setup`](https://github.com/joschi/setup-jdk/releases/tag/v2.5.1) is deprecated (also it still uses Node12 which is already deprecated by [GH](https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/))

Reply:
oki, I can update that

Reply:
PR updated